packages/js/src/utils/start-local-registry.ts

```diff
@@ -0,0 +1,68 @@
+import { execSync, fork } from 'child_process';
+
+/**
+ * This function is used to start a local registry for testing purposes.
+ * @param localRegistryTarget the target to run to start the local registry e.g. workspace:local-registry
+ * @param storage the storage location for the local registry
+ * @param verbose whether to log verbose output
+ */
+export async function startLocalRegistry({
+  localRegistryTarget,
+  storage,
+  verbose,
+}: {
+  localRegistryTarget: string;
+  storage?: string;
+  verbose?: boolean;
+}) {
```

Comment:
Is something like the following possible?

```ts
global.cleanupLocalRegistry = startLocalRegistry(...);
```

Reply:
changed